e2e/pages/app/infinite-scroll/standard/infinite-scroll.testpage.component.ts

```diff
@@ -34,7 +34,7 @@ export class InfiniteScrollTestPageComponent {
                     .filter((e) => this.isFilterMatch(e))
                     .slice(pageStart, pageStart + pageSize);
                 resolve(newItems);
-            }, 200);
+            }, 2000);
```

Comment:
What was the reason for needing this change? We generally want to avoid timeouts etc.. in e2e tests, as if we have 10 tests, waiting 200ms this adds a 2 second delay, but bumping them up to 2 seconds delay could increase the test time to 20 seconds for 10 tests which we want to avoid as our e2e tests are already slow

Reply:
I just noticed that the set timeout was 2000 in the fullscreen test and 200 here. I can try reduce this on both tests to 500. I think the problem might have been linked to Line17 `debouncedFilterText = this.filterText.pipe(debounceTime(500));` Because the load request isnt called twice now, having a timeout of less than 500 may have caused it to fail?

Reply:
I think in this case we should reduce the debounce time to a really low value (in both fullscreen and non fullscreen).

We are programmatically setting the input value so we won't need much of a debounce as that is to prevent unnecessary work if the person is slow at typing, which protractor will not be. Any e2e test speed improvements is a great.